Sources/Core/Presenter/Channel/ChannelPresenter.swift

```diff
@@ -203,3 +203,19 @@ extension ChannelPresenter {
         rx.markReadIfPossible().bindOnce(to: completion)
     }
 }
+
+// MARK: - Unused Channel for Atomic
+
+extension Channel {
```

Comment:
Can't we make this `fileprivate extension` since (if?) it's only used here?

Reply:
Unfortunately, we can't, it used in public space.

Reply:
At least I moved it from Low-Level Client to Core level.